tests/searchfox-config.json

```
@@ -15,7 +15,8 @@
       "git_path": "$MOZSEARCH_PATH",
       "git_blame_path": "$WORKING/searchfox/blame",
       "codesearch_path": "$WORKING/searchfox/livegrep.idx",
-      "codesearch_port": 8081
+      "codesearch_port": 8081,
+      "scip_subtrees": {}
```

To have the Rust indexing back in searchfox too:
```suggestion
      "scip_subtrees": {
        "rust": {
          "scip_index_path": "$WORKING/searchfox/objdir/tools/rust.scip",
          "subtree_root": "tools"
        }
      }
```